src/main/java/ch/njol/skript/conditions/CondIsDivisibleBy.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Checker;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Divisible By")
+@Description("Check if a number is divisible by another number.")
+@Examples({"if 5 is divisible by 5:",
+		"if 1964903306 is divisible by 982451653:",
+		"if 11 cannot be divided by 10:",
+		"if 10007 cannnot be divided by 10007:"})
+@Since("INSERT VERSION")
+public class CondIsDivisibleBy extends Condition {
+
+	static {
+		Skript.registerCondition(CondIsDivisibleBy.class,
+			"%numbers% (is|are) divisible by %number%",
+			"%numbers% (isn't|is not|aren't|are not) divisible by %number%",
+			"%numbers% can be [evenly] divided by %number%",
```

Comment:
should evenly be optional?

Reply:
Eh it should be left optional in my opinion.

Reply:
how come? it is a bit confusing to me that `if 2 is divisible by 3` is false. it is divisible, but its not evenly divisible

Reply:
Valid point